Legacy streams that play in 1.3.5 do not play in 1.4.9

### What version of Hls.js are you using?

1.4.9

### What browser (including version) are you using?

Chrome 114.0.5735.198 (Official Build) (arm64), Safari 16.5.1 (18615.2.9.11.7) 

### What OS (including version) are you using?

macOS Ventura Version 13.4.1 (22F82)

### Test stream

Can provide via DM

### Configuration

```JavaScript
{
  "debug": true,
  "enableWorker": true,
  "lowLatencyMode": true,
  "backBufferLength": 90
}
```


### Additional player setup steps

_No response_

### Checklist

- [X] The issue observed is not already reported by searching on Github under https://github.com/video-dev/hls.js/issues
- [X] The issue occurs in the stable client (latest release) on https://hlsjs.video-dev.org/demo and not just on my page
- [X] The issue occurs in the latest client (main branch) on https://hlsjs-dev.video-dev.org/demo and not just on my page
- [X] The stream has correct Access-Control-Allow-Origin headers (CORS)
- [X] There are no network errors such as 404s in the browser console when trying to play the stream

### Steps to reproduce

1. Load the stream in the Hls.js demo player.

### Expected behaviour

The VOD stream should play for the full duration.

### What actually happened?

Playback stops after a few seconds, `fragParsingError`.
```
base-stream-controller.ts:1530 [warn] > fragParsingError reached or exceeded max retry (6)
main.js:734 Error event: 
{type: 'mediaError', details: 'fragParsingError', fatal: true, error: Error: Found no media in fragment 15 of level 0 resetting transmuxer to fallback to playlist timing…, frag: Fragment, …}
main.js:847 Fatal error : fragParsingError
```


### Console output

```shell
15:58:00.697 main.js:339 Using Hls.js config: {debug: true, enableWorker: true, lowLatencyMode: true, backBufferLength: 90}
15:58:00.697 logger.ts:74 [log] > Debug logs enabled for "Hls instance" in hls.js version 1.4.9
15:58:00.701 hls.ts:410 [log] > stopLoad
15:58:00.701 hls.ts:379 [log] > loadSource:https://...master.m3u8
15:58:00.702 stream-controller.ts:569 [log] > [stream-controller]: Trigger BUFFER_RESET
15:58:00.702 hls.ts:351 [log] > attachMedia
15:58:00.720 level-controller.ts:269 [log] > [level-controller]: manifest loaded, 6 level(s) found, first bitrate: 930431
15:58:00.720 buffer-controller.ts:148 [log] > 2 bufferCodec event(s) expected
15:58:00.722 hls.ts:400 [log] > startLoad(-1)
15:58:00.722 level-controller.ts:351 [log] > [level-controller]: Switching to level 4 from level -1
15:58:00.722 audio-track-controller.ts:138 [log] > [audio-track-controller]: Updating audio tracks, 1 track(s) found in group:audio0
15:58:00.723 audio-track-controller.ts:195 [log] > [audio-track-controller]: Switching to audio-track 0 "English" lang:undefined group:audio0
15:58:00.723 base-stream-controller.ts:1615 [log] > [audio-stream-controller]: Reset loading state
15:58:00.723 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: STOPPED->IDLE
15:58:00.723 subtitle-track-controller.ts:217 [log] > [subtitle-track-controller]: Updating subtitle tracks, 1 track(s) found in "subs0" group-id
15:58:00.724 level-controller.ts:520 [log] > [level-controller]: Loading level index 4 with URI 1/1 https://.../index-f1-v1.m3u8
15:58:00.724 base-stream-controller.ts:1754 [log] > [stream-controller]: STOPPED->IDLE
15:58:00.724 audio-track-controller.ts:273 [log] > [audio-track-controller]: loading audio-track playlist 0 "English" lang:undefined group:audio0
15:58:00.725 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: IDLE->STOPPED
15:58:00.725 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: STOPPED->WAITING_TRACK
15:58:00.725 base-stream-controller.ts:1754 [log] > [subtitle-stream-controller]: STOPPED->IDLE
15:58:00.725 buffer-controller.ts:800 [log] > [buffer-controller]: Media source opened
15:58:00.725 subtitle-track-controller.ts:371 [log] > [subtitle-track-controller]: Switching to subtitle-track 0 "English" lang:en group:subs0
15:58:00.725 subtitle-track-controller.ts:292 [log] > [subtitle-track-controller]: Loading subtitle playlist for id 0
15:58:00.739 audio-track-controller.ts:90 [log] > [audio-track-controller]: audio-track 0 "English" lang:undefined group:audio0 loaded [1-32]
15:58:00.742 subtitle-track-controller.ts:173 [log] > [subtitle-track-controller]: subtitle track 0 loaded [1-32]
15:58:00.743 base-stream-controller.ts:727 [log] > [subtitle-stream-controller]: Loading fragment 1 cc: 0 of [1-32] track: 0, target: 0
15:58:00.743 base-stream-controller.ts:1754 [log] > [subtitle-stream-controller]: IDLE->FRAG_LOADING
15:58:00.745 stream-controller.ts:634 [log] > [stream-controller]: Level 4 loaded [1,32][part-32--1], cc [0, 0] duration:126.96
15:58:00.745 buffer-controller.ts:692 [log] > [buffer-controller]: Updating Media Source duration to 126.960
15:58:00.746 base-stream-controller.ts:629 [log] > [stream-controller]: Loading key for 1 of [1-32], level 4
15:58:00.746 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->KEY_LOADING
15:58:00.746 base-stream-controller.ts:727 [log] > [stream-controller]: Loading fragment 1 cc: 0 of [1-32] level: 4, target: 0
15:58:00.746 base-stream-controller.ts:1754 [log] > [stream-controller]: KEY_LOADING->FRAG_LOADING
15:58:00.747 audio-track-controller.ts:90 [log] > [audio-track-controller]: audio-track 0 "English" lang:undefined group:audio0 loaded [1-32]
15:58:00.747 audio-stream-controller.ts:497 [log] > [audio-stream-controller]: Track 0 loaded [1,32][part-32--1],duration:126.96
15:58:00.747 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: WAITING_TRACK->IDLE
15:58:00.747 base-stream-controller.ts:629 [log] > [audio-stream-controller]: Loading key for 1 of [1-32], track 0
15:58:00.747 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: IDLE->KEY_LOADING
15:58:00.747 base-stream-controller.ts:727 [log] > [audio-stream-controller]: Loading fragment 1 cc: 0 of [1-32] track: 0, target: 0
15:58:00.747 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: KEY_LOADING->FRAG_LOADING
///
15:58:00.759 base-stream-controller.ts:386 [log] > [subtitle-stream-controller]: Loaded fragment 1 of level 0
15:58:00.792 transmuxer-interface.ts:85 [log] > injecting Web Worker for "main"
15:58:00.795 transmuxer-interface.ts:227 [log] > [transmuxer-interface, main]: Starting new transmux session for sn: 1 p: -1 level: 4 id: 1
        discontinuity: true
        trackSwitch: true
        contiguous: false
        accurateTimeOffset: true
        timeOffset: 0
        initSegmentChange: true
15:58:00.795 base-stream-controller.ts:386 [log] > [stream-controller]: Loaded fragment 1 of level 4
15:58:00.903 blob:https://hlsjs.video-dev.org/ba2c0051-f62b-4843-a953-5cf9bf474848:542 [log] > Debug logs enabled for "main" in hls.js version 1.4.9
15:58:00.904 transmuxer-interface.ts:379 [log] > [decrypter]: JS AES decrypt
15:58:00.918 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: ISGenerated flag reset
15:58:00.918 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: initPTS & initDTS reset
15:58:00.918 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: reset next timestamp
15:58:00.924 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 1 of level 4
15:58:00.925 base-stream-controller.ts:1754 [log] > [stream-controller]: FRAG_LOADING->PARSING
15:58:00.925 stream-controller.ts:1288 [log] > [stream-controller]: Init video buffer, container:video/mp4, codecs[level/parsed]=[avc1.42c01f/avc1.42c01f]
15:58:00.928 audio-stream-controller.ts:128 [log] > [audio-stream-controller]: InitPTS for cc: 0 found from main: 9090
15:58:00.929 base-stream-controller.ts:1754 [log] > [subtitle-stream-controller]: FRAG_LOADING->IDLE
15:58:00.933 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSING->PARSED
15:58:01.038 transmuxer-interface.ts:85 [log] > injecting Web Worker for "audio"
15:58:01.041 transmuxer-interface.ts:227 [log] > [transmuxer-interface, audio]: Starting new transmux session for sn: 1 p: -1 level: 0 id: 1
        discontinuity: true
        trackSwitch: true
        contiguous: false
        accurateTimeOffset: false
        timeOffset: 0
        initSegmentChange: true
15:58:01.041 base-stream-controller.ts:386 [log] > [audio-stream-controller]: Loaded fragment 1 of level 0
15:58:01.067 blob:https://hlsjs.video-dev.org/fccd97b3-31ec-4be0-9cd3-678c26a42a0c:542 [log] > Debug logs enabled for "audio" in hls.js version 1.4.9
15:58:01.068 transmuxer-interface.ts:379 [log] > [decrypter]: JS AES decrypt
15:58:01.075 transmuxer-interface.ts:379 [log] > ADTS sync word found !
15:58:01.076 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: ISGenerated flag reset
15:58:01.076 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: initPTS & initDTS reset
15:58:01.076 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: reset next timestamp
15:58:01.076 transmuxer-interface.ts:379 [log] > manifest codec:mp4a.40.2, ADTS type:2, samplingIndex:7
15:58:01.076 transmuxer-interface.ts:379 [log] > parsed codec:mp4a.40.2, rate:22050, channels:1
15:58:01.077 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 1 of level 0
15:58:01.078 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: FRAG_LOADING->PARSING
15:58:01.080 audio-stream-controller.ts:848 [log] > [audio-stream-controller]: Init audio buffer, container:audio/mp4, codecs[parsed]=[mp4a.40.2]
15:58:01.080 buffer-controller.ts:765 [log] > [buffer-controller]: creating sourceBuffer(video/mp4;codecs=avc1.42c01f)
15:58:01.080 buffer-controller.ts:765 [log] > [buffer-controller]: creating sourceBuffer(audio/mp4;codecs=mp4a.40.2)
15:58:01.080 stream-controller.ts:847 [log] > [stream-controller]: Alternate track found, use video.buffered to schedule main fragment loading
15:58:01.086 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: PARSING->PARSED
15:58:01.103 base-stream-controller.ts:556 [log] > [audio-stream-controller]: Buffered audio sn: 1 of track 0 (frag:[0.000-4.040] > buffer:[0.000-4.040])
15:58:01.103 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: PARSED->IDLE
15:58:01.109 base-stream-controller.ts:556 [log] > [stream-controller]: Buffered main sn: 1 of level 4 (frag:[0.000-6.006] > buffer:[0.000-6.006])
15:58:01.110 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSED->IDLE
15:58:01.110 stream-controller.ts:261 [log] > [stream-controller]: Adapting to level 5 from level 4
15:58:01.110 level-controller.ts:351 [log] > [level-controller]: Switching to level 5 from level 4
15:58:01.110 level-controller.ts:520 [log] > [level-controller]: Loading level index 5 with URI 1/1 https://.../index-f6-v1.m3u8
15:58:01.111 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->WAITING_LEVEL
15:58:01.126 stream-controller.ts:634 [log] > [stream-controller]: Level 5 loaded [1,32][part-32--1], cc [0, 0] duration:126.96
15:58:01.127 base-stream-controller.ts:1754 [log] > [stream-controller]: WAITING_LEVEL->IDLE
15:58:01.127 base-stream-controller.ts:629 [log] > [stream-controller]: Loading key for 2 of [1-32], level 5
15:58:01.127 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->KEY_LOADING
15:58:01.127 base-stream-controller.ts:727 [log] > [stream-controller]: Loading fragment 2 cc: 0 of [1-32] level: 5, target: 6.006
15:58:01.127 base-stream-controller.ts:1754 [log] > [stream-controller]: KEY_LOADING->FRAG_LOADING
15:58:01.129 base-stream-controller.ts:629 [log] > [audio-stream-controller]: Loading key for 2 of [1-32], track 0
15:58:01.129 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: IDLE->KEY_LOADING
15:58:01.129 base-stream-controller.ts:727 [log] > [audio-stream-controller]: Loading fragment 2 cc: 0 of [1-32] track: 0, target: 4.04
15:58:01.129 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: KEY_LOADING->FRAG_LOADING
15:58:01.142 base-stream-controller.ts:386 [log] > [audio-stream-controller]: Loaded fragment 2 of level 0
15:58:01.143 transmuxer-interface.ts:227 [log] > [transmuxer-interface, main]: Starting new transmux session for sn: 2 p: -1 level: 5 id: 1
        discontinuity: false
        trackSwitch: true
        contiguous: false
        accurateTimeOffset: true
        timeOffset: 4
        initSegmentChange: false
15:58:01.143 base-stream-controller.ts:386 [log] > [stream-controller]: Loaded fragment 2 of level 5
15:58:01.153 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 2 of level 0
15:58:01.153 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: FRAG_LOADING->PARSING
15:58:01.157 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: PARSING->PARSED
15:58:01.158 favicon.ico:1     GET https://hlsjs.video-dev.org/favicon.ico 404
15:58:01.158 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: ISGenerated flag reset
15:58:01.158 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: initPTS & initDTS reset
15:58:01.158 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: reset next timestamp
15:58:01.166 base-stream-controller.ts:556 [log] > [audio-stream-controller]: Buffered audio sn: 2 of track 0 (frag:[4.040-8.034] > buffer:[0.000-8.034])
15:58:01.166 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: PARSED->IDLE
15:58:01.166 base-stream-controller.ts:629 [log] > [audio-stream-controller]: Loading key for 3 of [1-32], track 0
15:58:01.166 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: IDLE->KEY_LOADING
15:58:01.166 base-stream-controller.ts:727 [log] > [audio-stream-controller]: Loading fragment 3 cc: 0 of [1-32] track: 0, target: 8.034
15:58:01.166 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: KEY_LOADING->FRAG_LOADING
15:58:01.167 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 2 of level 5
15:58:01.167 base-stream-controller.ts:1754 [log] > [stream-controller]: FRAG_LOADING->PARSING
15:58:01.167 stream-controller.ts:1288 [log] > [stream-controller]: Init video buffer, container:video/mp4, codecs[level/parsed]=[avc1.42c01f/avc1.42c01f]
15:58:01.174 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSING->PARSED
15:58:01.174 base-stream-controller.ts:386 [log] > [audio-stream-controller]: Loaded fragment 3 of level 0
15:58:01.181 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 3 of level 0
15:58:01.181 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: FRAG_LOADING->PARSING
15:58:01.184 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: PARSING->PARSED
15:58:01.193 base-stream-controller.ts:556 [log] > [audio-stream-controller]: Buffered audio sn: 3 of track 0 (frag:[8.034-12.028] > buffer:[0.000-12.028])
15:58:01.193 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: PARSED->IDLE
15:58:01.199 base-stream-controller.ts:556 [log] > [stream-controller]: Buffered main sn: 2 of level 5 (frag:[6.006-12.012] > buffer:[0.000-12.012])
15:58:01.199 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSED->IDLE
15:58:01.199 base-stream-controller.ts:629 [log] > [stream-controller]: Loading key for 3 of [1-32], level 5
15:58:01.199 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->KEY_LOADING
15:58:01.199 base-stream-controller.ts:727 [log] > [stream-controller]: Loading fragment 3 cc: 0 of [1-32] level: 5, target: 12.012
15:58:01.199 base-stream-controller.ts:1754 [log] > [stream-controller]: KEY_LOADING->FRAG_LOADING
15:58:01.202 base-stream-controller.ts:386 [log] > [stream-controller]: Loaded fragment 3 of level 5
15:58:01.202 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 3 of level 5
15:58:01.202 base-stream-controller.ts:1754 [log] > [stream-controller]: FRAG_LOADING->PARSING
15:58:01.202 base-stream-controller.ts:1713 [warn] > [stream-controller]: Found no media in fragment 3 of level 0 resetting transmuxer to fallback to playlist timing
updateLevelTiming @ base-stream-controller.ts:1713
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
15:58:01.202 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSING->IDLE
15:58:01.202 error-controller.ts:461 [warn] > [warning]: switching to level 4 after fragParsingError
switchLevel @ error-controller.ts:461
sendAlternateToPenaltyBox @ error-controller.ts:433
onErrorOut @ error-controller.ts:404
emit @ index.js:203
emit @ hls.ts:287
trigger @ hls.ts:295
updateLevelTiming @ base-stream-controller.ts:1714
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
Show 1 more frame
15:58:01.203 level-controller.ts:351 [log] > [level-controller]: Switching to level 4 from level 5
15:58:01.203 main.js:734 Error event: {type: 'mediaError', details: 'fragParsingError', fatal: false, error: Error: Found no media in fragment 3 of level 0 resetting transmuxer to fallback to playlist timing
…, frag: Fragment, …}
(anonymous) @ main.js:734
emit @ index.js:203
emit @ hls.ts:287
trigger @ hls.ts:295
updateLevelTiming @ base-stream-controller.ts:1714
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
Show 1 more frame
15:58:01.203 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->PARSED
15:58:01.203 buffer-controller.ts:523 [warn] > Fragments must have at least one ElementaryStreamType set. type: main level: 5 sn: 3
onFragParsed @ buffer-controller.ts:523
emit @ index.js:203
emit @ hls.ts:287
trigger @ hls.ts:295
updateLevelTiming @ base-stream-controller.ts:1729
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
Show 1 more frame
15:58:01.203 buffer-controller.ts:921 [log] > [buffer-controller]: Blocking operation requested, but no SourceBuffers exist
15:58:01.204 base-stream-controller.ts:556 [log] > [stream-controller]: Buffered main sn: 3 of level 5 (frag:[NaN-NaN] > buffer:[0.000-12.012])
15:58:01.204 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSED->IDLE
15:58:01.204 base-stream-controller.ts:629 [log] > [stream-controller]: Loading key for 3 of [1-32], level 4
15:58:01.204 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->KEY_LOADING
15:58:01.204 base-stream-controller.ts:727 [log] > [stream-controller]: Loading fragment 3 cc: 0 of [1-32] level: 4, target: 12.012
15:58:01.204 base-stream-controller.ts:1754 [log] > [stream-controller]: KEY_LOADING->FRAG_LOADING
15:58:01.206 transmuxer-interface.ts:85 [log] > injecting Web Worker for "main"
15:58:01.208 transmuxer-interface.ts:227 [log] > [transmuxer-interface, main]: Starting new transmux session for sn: 3 p: -1 level: 4 id: 1
        discontinuity: true
        trackSwitch: true
        contiguous: false
        accurateTimeOffset: true
        timeOffset: 10.00601111111111
        initSegmentChange: true
15:58:01.208 base-stream-controller.ts:386 [log] > [stream-controller]: Loaded fragment 3 of level 4
15:58:01.225 base-stream-controller.ts:629 [log] > [audio-stream-controller]: Loading key for 4 of [1-32], track 0
15:58:01.225 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: IDLE->KEY_LOADING
15:58:01.225 base-stream-controller.ts:727 [log] > [audio-stream-controller]: Loading fragment 4 cc: 0 of [1-32] track: 0, target: 12.028
15:58:01.225 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: KEY_LOADING->FRAG_LOADING
15:58:01.226 base-stream-controller.ts:727 [log] > [subtitle-stream-controller]: Loading fragment 2 cc: 0 of [1-32] track: 0, target: 4
15:58:01.226 base-stream-controller.ts:1754 [log] > [subtitle-stream-controller]: IDLE->FRAG_LOADING
15:58:01.227 base-stream-controller.ts:386 [log] > [audio-stream-controller]: Loaded fragment 4 of level 0
15:58:01.231 base-stream-controller.ts:386 [log] > [subtitle-stream-controller]: Loaded fragment 2 of level 0
15:58:01.232 base-stream-controller.ts:1754 [log] > [subtitle-stream-controller]: FRAG_LOADING->IDLE
15:58:01.232 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 4 of level 0
15:58:01.232 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: FRAG_LOADING->PARSING
15:58:01.235 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: PARSING->PARSED
15:58:01.240 base-stream-controller.ts:556 [log] > [audio-stream-controller]: Buffered audio sn: 4 of track 0 (frag:[12.028-16.022] > buffer:[0.000-16.022])
15:58:01.240 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: PARSED->IDLE
15:58:01.242 blob:https://hlsjs.video-dev.org/f2cad414-8cda-431c-b0af-eb10c69299b3:542 [log] > Debug logs enabled for "main" in hls.js version 1.4.9
15:58:01.242 transmuxer-interface.ts:379 [log] > [decrypter]: JS AES decrypt
15:58:01.243 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: ISGenerated flag reset
15:58:01.243 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: initPTS & initDTS reset
15:58:01.243 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: reset next timestamp
15:58:01.244 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 3 of level 4
15:58:01.244 base-stream-controller.ts:1754 [log] > [stream-controller]: FRAG_LOADING->PARSING
15:58:01.244 base-stream-controller.ts:1713 [warn] > [stream-controller]: Found no media in fragment 3 of level 0 resetting transmuxer to fallback to playlist timing
updateLevelTiming @ base-stream-controller.ts:1713
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
15:58:01.244 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSING->IDLE
15:58:01.244 error-controller.ts:461 [warn] > [warning]: switching to level 3 after fragParsingError
switchLevel @ error-controller.ts:461
sendAlternateToPenaltyBox @ error-controller.ts:433
onErrorOut @ error-controller.ts:404
emit @ index.js:203
emit @ hls.ts:287
trigger @ hls.ts:295
updateLevelTiming @ base-stream-controller.ts:1714
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
Show 1 more frame
15:58:01.244 level-controller.ts:351 [log] > [level-controller]: Switching to level 3 from level 4
15:58:01.244 level-controller.ts:520 [log] > [level-controller]: Loading level index 3 with URI 1/1 https://.../index-f5-v1.m3u8
15:58:01.245 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->WAITING_LEVEL
15:58:01.245 main.js:734 Error event: {type: 'mediaError', details: 'fragParsingError', fatal: false, error: Error: Found no media in fragment 3 of level 0 resetting transmuxer to fallback to playlist timing
…, frag: Fragment, …}
///
15:58:01.957 main.js:734 Error event: {type: 'mediaError', details: 'fragParsingError', fatal: false, error: Error: Found no media in fragment 12 of level 0 resetting transmuxer to fallback to playlist timing…, frag: Fragment, …}
(anonymous) @ main.js:734
emit @ index.js:203
emit @ hls.ts:287
trigger @ hls.ts:295
updateLevelTiming @ base-stream-controller.ts:1714
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
Show 1 more frame
15:58:01.957 base-stream-controller.ts:1754 [log] > [stream-controller]: WAITING_LEVEL->PARSED
15:58:01.957 buffer-controller.ts:523 [warn] > Fragments must have at least one ElementaryStreamType set. type: main level: 1 sn: 12
onFragParsed @ buffer-controller.ts:523
emit @ index.js:203
emit @ hls.ts:287
trigger @ hls.ts:295
updateLevelTiming @ base-stream-controller.ts:1729
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
Show 1 more frame
15:58:01.957 buffer-controller.ts:921 [log] > [buffer-controller]: Blocking operation requested, but no SourceBuffers exist
15:58:01.958 base-stream-controller.ts:556 [log] > [stream-controller]: Buffered main sn: 12 of level 1 (frag:[NaN-NaN] > buffer:[0.000-48.048])
15:58:01.958 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSED->IDLE
15:58:01.958 level-controller.ts:351 [log] > [level-controller]: Switching to level 0 from level 0
15:58:01.958 level-controller.ts:520 [log] > [level-controller]: Loading level index 0 with URI 1/1 https://.../index-f2-v1.m3u8
15:58:01.958 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->WAITING_LEVEL
15:58:01.974 stream-controller.ts:634 [log] > [stream-controller]: Level 0 loaded [1,32][part-32--1], cc [0, 0] duration:126.96
15:58:01.974 base-stream-controller.ts:1754 [log] > [stream-controller]: WAITING_LEVEL->IDLE
15:58:01.974 base-stream-controller.ts:629 [log] > [stream-controller]: Loading key for 13 of [1-32], level 0
15:58:01.974 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->KEY_LOADING
15:58:01.974 base-stream-controller.ts:727 [log] > [stream-controller]: Loading fragment 13 cc: 0 of [1-32] level: 0, target: 48.048
15:58:01.974 base-stream-controller.ts:1754 [log] > [stream-controller]: KEY_LOADING->FRAG_LOADING
15:58:01.987 transmuxer-interface.ts:85 [log] > injecting Web Worker for "main"
15:58:01.989 transmuxer-interface.ts:227 [log] > [transmuxer-interface, main]: Starting new transmux session for sn: 13 p: -1 level: 0 id: 1
        discontinuity: true
        trackSwitch: true
        contiguous: false
        accurateTimeOffset: true
        timeOffset: 48
        initSegmentChange: true
15:58:01.990 base-stream-controller.ts:386 [log] > [stream-controller]: Loaded fragment 13 of level 0
15:58:02.014 blob:https://hlsjs.video-dev.org/1a8721c8-24a7-49fa-9900-6a6737fe9817:542 [log] > Debug logs enabled for "main" in hls.js version 1.4.9
15:58:02.015 transmuxer-interface.ts:379 [log] > [decrypter]: JS AES decrypt
15:58:02.024 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: ISGenerated flag reset
15:58:02.024 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: initPTS & initDTS reset
15:58:02.024 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: reset next timestamp
15:58:02.030 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 13 of level 0
15:58:02.030 base-stream-controller.ts:1754 [log] > [stream-controller]: FRAG_LOADING->PARSING
15:58:02.030 stream-controller.ts:1288 [log] > [stream-controller]: Init video buffer, container:video/mp4, codecs[level/parsed]=[avc1.42c00d/avc1.42c00d]
15:58:02.039 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSING->PARSED
15:58:02.054 base-stream-controller.ts:556 [log] > [stream-controller]: Buffered main sn: 13 of level 0 (frag:[48.048-54.054] > buffer:[0.000-54.054])
15:58:02.054 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSED->IDLE
15:58:02.055 stream-controller.ts:261 [log] > [stream-controller]: Adapting to level 5 from level 0
15:58:02.055 level-controller.ts:351 [log] > [level-controller]: Switching to level 5 from level 0
15:58:02.055 base-stream-controller.ts:629 [log] > [stream-controller]: Loading key for 13 of [1-32], level 5
15:58:02.055 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->KEY_LOADING
15:58:02.055 base-stream-controller.ts:727 [log] > [stream-controller]: Loading fragment 13 cc: 0 of [1-32] level: 5, target: 54.054
15:58:02.055 base-stream-controller.ts:1754 [log] > [stream-controller]: KEY_LOADING->FRAG_LOADING
15:58:02.057 transmuxer-interface.ts:227 [log] > [transmuxer-interface, main]: Starting new transmux session for sn: 13 p: -1 level: 5 id: 1
        discontinuity: false
        trackSwitch: true
        contiguous: false
        accurateTimeOffset: true
        timeOffset: 52.04805555555556
        initSegmentChange: false
15:58:02.058 base-stream-controller.ts:386 [log] > [stream-controller]: Loaded fragment 13 of level 5
15:58:02.073 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: ISGenerated flag reset
15:58:02.073 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: initPTS & initDTS reset
15:58:02.073 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: reset next timestamp
15:58:02.079 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 13 of level 5
15:58:02.079 base-stream-controller.ts:1754 [log] > [stream-controller]: FRAG_LOADING->PARSING
15:58:02.079 stream-controller.ts:1288 [log] > [stream-controller]: Init video buffer, container:video/mp4, codecs[level/parsed]=[avc1.42c01f/avc1.42c01f]
15:58:02.088 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSING->PARSED
15:58:02.104 base-stream-controller.ts:556 [log] > [stream-controller]: Buffered main sn: 13 of level 5 (frag:[48.048-54.054] > buffer:[0.000-54.054])
15:58:02.104 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSED->IDLE
15:58:02.104 base-stream-controller.ts:629 [log] > [stream-controller]: Loading key for 14 of [1-32], level 5
15:58:02.104 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->KEY_LOADING
15:58:02.104 base-stream-controller.ts:727 [log] > [stream-controller]: Loading fragment 14 cc: 0 of [1-32] level: 5, target: 54.054
15:58:02.104 base-stream-controller.ts:1754 [log] > [stream-controller]: KEY_LOADING->FRAG_LOADING
15:58:02.106 base-stream-controller.ts:386 [log] > [stream-controller]: Loaded fragment 14 of level 5
15:58:02.119 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 14 of level 5
15:58:02.119 base-stream-controller.ts:1754 [log] > [stream-controller]: FRAG_LOADING->PARSING
15:58:02.124 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSING->PARSED
15:58:02.132 base-stream-controller.ts:556 [log] > [stream-controller]: Buffered main sn: 14 of level 5 (frag:[54.054-60.060] > buffer:[0.000-60.060])
15:58:02.132 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSED->IDLE
15:58:02.132 base-stream-controller.ts:629 [log] > [stream-controller]: Loading key for 15 of [1-32], level 5
15:58:02.132 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->KEY_LOADING
15:58:02.132 base-stream-controller.ts:727 [log] > [stream-controller]: Loading fragment 15 cc: 0 of [1-32] level: 5, target: 60.06
15:58:02.132 base-stream-controller.ts:1754 [log] > [stream-controller]: KEY_LOADING->FRAG_LOADING
15:58:02.133 base-stream-controller.ts:629 [log] > [audio-stream-controller]: Loading key for 14 of [1-32], track 0
15:58:02.133 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: IDLE->KEY_LOADING
15:58:02.133 base-stream-controller.ts:727 [log] > [audio-stream-controller]: Loading fragment 14 cc: 0 of [1-32] track: 0, target: 52.013
15:58:02.133 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: KEY_LOADING->FRAG_LOADING
15:58:02.134 base-stream-controller.ts:386 [log] > [stream-controller]: Loaded fragment 15 of level 5
15:58:02.135 base-stream-controller.ts:386 [log] > [audio-stream-controller]: Loaded fragment 14 of level 0
15:58:02.135 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 15 of level 5
15:58:02.135 base-stream-controller.ts:1754 [log] > [stream-controller]: FRAG_LOADING->PARSING
15:58:02.135 base-stream-controller.ts:1713 [warn] > [stream-controller]: Found no media in fragment 15 of level 0 resetting transmuxer to fallback to playlist timing
updateLevelTiming @ base-stream-controller.ts:1713
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
15:58:02.135 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSING->IDLE
15:58:02.135 error-controller.ts:461 [warn] > [warning]: switching to level 0 after fragParsingError
switchLevel @ error-controller.ts:461
sendAlternateToPenaltyBox @ error-controller.ts:433
onErrorOut @ error-controller.ts:404
emit @ index.js:203
emit @ hls.ts:287
trigger @ hls.ts:295
updateLevelTiming @ base-stream-controller.ts:1714
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
Show 1 more frame
15:58:02.135 level-controller.ts:351 [log] > [level-controller]: Switching to level 0 from level 5
15:58:02.135 main.js:734 Error event: {type: 'mediaError', details: 'fragParsingError', fatal: false, error: Error: Found no media in fragment 15 of level 0 resetting transmuxer to fallback to playlist timing…, frag: Fragment, …}
(anonymous) @ main.js:734
emit @ index.js:203
emit @ hls.ts:287
trigger @ hls.ts:295
updateLevelTiming @ base-stream-controller.ts:1714
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
Show 1 more frame
15:58:02.141 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->PARSED
15:58:02.142 buffer-controller.ts:523 [warn] > Fragments must have at least one ElementaryStreamType set. type: main level: 5 sn: 15
onFragParsed @ buffer-controller.ts:523
emit @ index.js:203
emit @ hls.ts:287
trigger @ hls.ts:295
updateLevelTiming @ base-stream-controller.ts:1729
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
Show 1 more frame
15:58:02.142 buffer-controller.ts:921 [log] > [buffer-controller]: Blocking operation requested, but no SourceBuffers exist
15:58:02.142 base-stream-controller.ts:556 [log] > [stream-controller]: Buffered main sn: 15 of level 5 (frag:[NaN-NaN] > buffer:[0.000-60.060])
15:58:02.142 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSED->IDLE
15:58:02.142 base-stream-controller.ts:629 [log] > [stream-controller]: Loading key for 15 of [1-32], level 0
15:58:02.142 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->KEY_LOADING
15:58:02.142 base-stream-controller.ts:727 [log] > [stream-controller]: Loading fragment 15 cc: 0 of [1-32] level: 0, target: 60.06
15:58:02.142 base-stream-controller.ts:1754 [log] > [stream-controller]: KEY_LOADING->FRAG_LOADING
15:58:02.143 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 14 of level 0
15:58:02.143 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: FRAG_LOADING->PARSING
15:58:02.148 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: PARSING->PARSED
15:58:02.152 transmuxer-interface.ts:85 [log] > injecting Web Worker for "main"
15:58:02.154 transmuxer-interface.ts:227 [log] > [transmuxer-interface, main]: Starting new transmux session for sn: 15 p: -1 level: 0 id: 1
        discontinuity: true
        trackSwitch: true
        contiguous: false
        accurateTimeOffset: true
        timeOffset: 58.05405555555556
        initSegmentChange: true
15:58:02.154 base-stream-controller.ts:386 [log] > [stream-controller]: Loaded fragment 15 of level 0
15:58:02.168 base-stream-controller.ts:556 [log] > [audio-stream-controller]: Buffered audio sn: 14 of track 0 (frag:[52.013-56.007] > buffer:[0.000-56.006])
15:58:02.168 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: PARSED->IDLE
15:58:02.168 base-stream-controller.ts:629 [log] > [audio-stream-controller]: Loading key for 15 of [1-32], track 0
15:58:02.168 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: IDLE->KEY_LOADING
15:58:02.168 base-stream-controller.ts:727 [log] > [audio-stream-controller]: Loading fragment 15 cc: 0 of [1-32] track: 0, target: 56.007
15:58:02.168 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: KEY_LOADING->FRAG_LOADING
15:58:02.172 base-stream-controller.ts:386 [log] > [audio-stream-controller]: Loaded fragment 15 of level 0
15:58:02.173 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 15 of level 0
15:58:02.173 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: FRAG_LOADING->PARSING
15:58:02.177 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: PARSING->PARSED
15:58:02.185 base-stream-controller.ts:556 [log] > [audio-stream-controller]: Buffered audio sn: 15 of track 0 (frag:[56.007-60.000] > buffer:[0.000-60.000])
15:58:02.185 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: PARSED->IDLE
15:58:02.185 base-stream-controller.ts:629 [log] > [audio-stream-controller]: Loading key for 16 of [1-32], track 0
15:58:02.185 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: IDLE->KEY_LOADING
15:58:02.185 base-stream-controller.ts:727 [log] > [audio-stream-controller]: Loading fragment 16 cc: 0 of [1-32] track: 0, target: 60
15:58:02.185 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: KEY_LOADING->FRAG_LOADING
15:58:02.188 base-stream-controller.ts:386 [log] > [audio-stream-controller]: Loaded fragment 16 of level 0
15:58:02.188 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 16 of level 0
15:58:02.188 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: FRAG_LOADING->PARSING
15:58:02.192 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: PARSING->PARSED
15:58:02.193 blob:https://hlsjs.video-dev.org/6ffabce9-fbc9-4f4a-a32d-7f6945c837a7:542 [log] > Debug logs enabled for "main" in hls.js version 1.4.9
15:58:02.200 base-stream-controller.ts:556 [log] > [audio-stream-controller]: Buffered audio sn: 16 of track 0 (frag:[60.000-64.041] > buffer:[0.000-64.041])
15:58:02.200 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: PARSED->IDLE
15:58:02.201 transmuxer-interface.ts:379 [log] > [decrypter]: JS AES decrypt
15:58:02.201 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: ISGenerated flag reset
15:58:02.201 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: initPTS & initDTS reset
15:58:02.201 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: reset next timestamp
15:58:02.201 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 15 of level 0
15:58:02.202 base-stream-controller.ts:1754 [log] > [stream-controller]: FRAG_LOADING->PARSING
15:58:02.202 base-stream-controller.ts:1713 [warn] > [stream-controller]: Found no media in fragment 15 of level 0 resetting transmuxer to fallback to playlist timing
updateLevelTiming @ base-stream-controller.ts:1713
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
15:58:02.202 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSING->IDLE
15:58:02.202 main.js:734 Error event: {type: 'mediaError', details: 'fragParsingError', fatal: false, error: Error: Found no media in fragment 15 of level 0 resetting transmuxer to fallback to playlist timing…, frag: Fragment, …}
(anonymous) @ main.js:734
emit @ index.js:203
emit @ hls.ts:287
trigger @ hls.ts:295
updateLevelTiming @ base-stream-controller.ts:1714
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
Show 1 more frame
15:58:02.202 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->PARSED
15:58:02.203 buffer-controller.ts:523 [warn] > Fragments must have at least one ElementaryStreamType set. type: main level: 0 sn: 15
onFragParsed @ buffer-controller.ts:523
emit @ index.js:203
emit @ hls.ts:287
trigger @ hls.ts:295
updateLevelTiming @ base-stream-controller.ts:1729
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
Show 1 more frame
15:58:02.203 buffer-controller.ts:921 [log] > [buffer-controller]: Blocking operation requested, but no SourceBuffers exist
15:58:02.203 base-stream-controller.ts:556 [log] > [stream-controller]: Buffered main sn: 15 of level 0 (frag:[NaN-NaN] > buffer:[0.000-60.060])
15:58:02.203 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSED->IDLE
15:58:02.203 stream-controller.ts:261 [log] > [stream-controller]: Adapting to level 5 from level 0
15:58:02.203 level-controller.ts:351 [log] > [level-controller]: Switching to level 5 from level 0
15:58:02.203 base-stream-controller.ts:727 [log] > [stream-controller]: Loading fragment 15 cc: 0 of [1-32] level: 5, target: 60.06
15:58:02.203 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->FRAG_LOADING
15:58:02.205 transmuxer-interface.ts:85 [log] > injecting Web Worker for "main"
15:58:02.207 transmuxer-interface.ts:227 [log] > [transmuxer-interface, main]: Starting new transmux session for sn: 15 p: -1 level: 5 id: 1
        discontinuity: true
        trackSwitch: true
        contiguous: false
        accurateTimeOffset: true
        timeOffset: 60.060066666666664
        initSegmentChange: true
15:58:02.207 base-stream-controller.ts:386 [log] > [stream-controller]: Loaded fragment 15 of level 5
15:58:02.225 base-stream-controller.ts:727 [log] > [subtitle-stream-controller]: Loading fragment 4 cc: 0 of [1-32] track: 0, target: 12
15:58:02.225 base-stream-controller.ts:1754 [log] > [subtitle-stream-controller]: IDLE->FRAG_LOADING
15:58:02.227 base-stream-controller.ts:386 [log] > [subtitle-stream-controller]: Loaded fragment 4 of level 0
15:58:02.227 base-stream-controller.ts:1754 [log] > [subtitle-stream-controller]: FRAG_LOADING->IDLE
15:58:02.231 blob:https://hlsjs.video-dev.org/a2800159-a030-4509-9b04-848912f25d3d:542 [log] > Debug logs enabled for "main" in hls.js version 1.4.9
15:58:02.231 transmuxer-interface.ts:379 [log] > [decrypter]: JS AES decrypt
15:58:02.232 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: ISGenerated flag reset
15:58:02.232 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: initPTS & initDTS reset
15:58:02.232 transmuxer-interface.ts:379 [log] > [mp4-remuxer]: reset next timestamp
15:58:02.233 transmuxer-interface.ts:379 [log] > [transmuxer.ts]: Flushed fragment 15 of level 5
15:58:02.233 base-stream-controller.ts:1754 [log] > [stream-controller]: FRAG_LOADING->PARSING
15:58:02.233 base-stream-controller.ts:1713 [warn] > [stream-controller]: Found no media in fragment 15 of level 0 resetting transmuxer to fallback to playlist timing
updateLevelTiming @ base-stream-controller.ts:1713
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
15:58:02.233 base-stream-controller.ts:1754 [log] > [stream-controller]: PARSING->IDLE
15:58:02.233 base-stream-controller.ts:1530 [warn] > fragParsingError reached or exceeded max retry (6)
onFragmentOrKeyLoadError @ base-stream-controller.ts:1530
onError @ stream-controller.ts:897
emit @ index.js:203
emit @ hls.ts:287
trigger @ hls.ts:295
updateLevelTiming @ base-stream-controller.ts:1714
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
Show 1 more frame
15:58:02.233 hls.ts:410 [log] > stopLoad
15:58:02.233 base-stream-controller.ts:1754 [log] > [stream-controller]: IDLE->STOPPED
15:58:02.233 base-stream-controller.ts:1754 [log] > [audio-stream-controller]: IDLE->STOPPED
15:58:02.233 base-stream-controller.ts:1754 [log] > [subtitle-stream-controller]: IDLE->STOPPED
15:58:02.233 main.js:734 Error event: {type: 'mediaError', details: 'fragParsingError', fatal: true, error: Error: Found no media in fragment 15 of level 0 resetting transmuxer to fallback to playlist timing…, frag: Fragment, …}
(anonymous) @ main.js:734
emit @ index.js:203
emit @ hls.ts:287
trigger @ hls.ts:295
updateLevelTiming @ base-stream-controller.ts:1714
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
Show 1 more frame
15:58:02.234 main.js:847 Fatal error : fragParsingError
(anonymous) @ main.js:847
emit @ index.js:203
emit @ hls.ts:287
trigger @ hls.ts:295
updateLevelTiming @ base-stream-controller.ts:1714
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
Show 1 more frame
15:58:02.235 hls.ts:431 [log] > recoverMediaError
15:58:02.235 hls.ts:360 [log] > detachMedia
15:58:02.235 buffer-controller.ts:175 [log] > [buffer-controller]: media source detaching
15:58:02.237 subtitle-track-controller.ts:371 [log] > [subtitle-track-controller]: Switching to subtitle-track -1
15:58:02.238 hls.ts:351 [log] > attachMedia
15:58:02.238 base-stream-controller.ts:1754 [log] > [stream-controller]: STOPPED->PARSED
15:58:02.238 buffer-controller.ts:523 [warn] > Fragments must have at least one ElementaryStreamType set. type: main level: 5 sn: 15
onFragParsed @ buffer-controller.ts:523
emit @ index.js:203
emit @ hls.ts:287
trigger @ hls.ts:295
updateLevelTiming @ base-stream-controller.ts:1729
_handleTransmuxerFlush @ base-stream-controller.ts:854
onWorkerMessage @ transmuxer-interface.ts:372
TransmuxerInterface.onwmsg @ transmuxer-interface.ts:88
Show 1 more frame
```


### Chrome media internals output

_No response_

--- a/src/controller/base-stream-controller.ts
+++ b/src/controller/base-stream-controller.ts
@@ -1708,8 +1708,15 @@ export default class BaseStreamController
       level.fragmentError = 0;
     } else if (this.transmuxer?.error === null) {
       const error = new Error(
-        `Found no media in fragment ${frag.sn} of level ${level.id} resetting transmuxer to fallback to playlist timing`
+        `Found no media in fragment ${frag.sn} of level ${frag.level} resetting transmuxer to fallback to playlist timing`
       );
+      if (level.fragmentError === 0) {
+        // Mark and track the odd empty segment as a gap to avoid reloading
+        level.fragmentError++;
+        frag.gap = true;
+        this.fragmentTracker.removeFragment(frag);
+        this.fragmentTracker.fragBuffered(frag, true);
+      }
       this.warn(error.message);
       this.hls.trigger(Events.ERROR, {
         type: ErrorTypes.MEDIA_ERROR,

--- a/src/controller/error-controller.ts
+++ b/src/controller/error-controller.ts
@@ -128,8 +128,17 @@ export default class ErrorController implements NetworkComponentAPI {
       case ErrorDetails.KEY_LOAD_TIMEOUT:
         data.errorAction = this.getFragRetryOrSwitchAction(data);
         return;
-      case ErrorDetails.FRAG_GAP:
       case ErrorDetails.FRAG_PARSING_ERROR:
+        // ignore empty segment errors marked as gap
+        if (data.frag?.gap) {
+          data.errorAction = {
+            action: NetworkErrorAction.DoNothing,
+            flags: ErrorActionFlags.None,
+          };
+          return;
+        }
+      // falls through
+      case ErrorDetails.FRAG_GAP:
       case ErrorDetails.FRAG_DECRYPT_ERROR: {
         // Switch level if possible, otherwise allow retry count to reach max error retries
         data.errorAction = this.getFragRetryOrSwitchAction(data);

--- a/src/controller/fragment-tracker.ts
+++ b/src/controller/fragment-tracker.ts
@@ -179,13 +179,13 @@ export class FragmentTracker implements ComponentAPI {
   public detectPartialFragments(data: FragBufferedData) {
     const timeRanges = this.timeRanges;
     const { frag, part } = data;
-    if (!timeRanges || frag.sn === 'initSegment' || frag.gap) {
+    if (!timeRanges || frag.sn === 'initSegment') {
       return;
     }
 
     const fragKey = getFragmentKey(frag);
     const fragmentEntity = this.fragments[fragKey];
-    if (!fragmentEntity) {
+    if (!fragmentEntity || (fragmentEntity.buffered && frag.gap)) {
       return;
     }
     const isFragHint = !frag.relurl;

--- a/src/controller/fragment-tracker.ts
+++ b/src/controller/fragment-tracker.ts
@@ -179,7 +179,7 @@ export class FragmentTracker implements ComponentAPI {
   public detectPartialFragments(data: FragBufferedData) {
     const timeRanges = this.timeRanges;
     const { frag, part } = data;
-    if (!timeRanges || frag.sn === 'initSegment') {
+    if (!timeRanges || frag.sn === 'initSegment' || frag.gap) {
       return;
     }
 

